integrationtests/testground/README.md

```diff
@@ -0,0 +1,55 @@
+Here contains test plans to be run on [Testground](https://docs.testground.ai/).
+
+# Setup
+
+```
+git clone https://github.com/testground/testground.git
+cd testground
+#  compile Testground and all related dependencies
+make install
+#  start the Testground daemon, listening by default on localhost:8042
+testground daemon
+```
+
+Run below from another terminal window. This will add tests here to the testground home directory, which is `$HOME/testground` by default, unless you explicitly define `$TESTGROUND_HOME`.
+```
+testground plan import --from . --name go-threads
+```
+
+# Run
+
+Then you can run the prepared composition plans as follows:
+```
+#  This one sticks to an early version of go-threads
+testground run composition --wait -f baseline-docker.toml
+#  This one always test against the current head of go-threads
+testground run composition --wait -f head-docker.toml
+#  This one uses whatever version specified by go.mod in the current directory. Do check/update dependencies before running this.
+testground run composition --wait -f current-docker.toml
+#  This one is good for local test while developing.
+testground run composition --wait -f current-exec.toml
+```
+
+You can also run individual test case and pass test parameters in command line. For example, below builds the test as native executables, runs 2 instances, with very verbose logs printed to the console. See manifest.toml for all test parameters.
+```
+testground run single --wait -p go-threads -t sync-threads -b exec:go -r local:exec -i 2 -tp verbose=2
+```
+
+# Analysize
```

Comment:
Still need to figure out how to access the Grafana and InfluxDB containers started along with testground daemon, but that's most useful in hosted environment, so we can skip atm.